src/lifx.d.ts

```diff
@@ -403,137 +356,15 @@ export const constants: {
   TAGGED_BIT: number;
   ZONE_INDEX_MAXIMUM_VALUE: number;
   ZONE_INDEX_MINIMUM_VALUE: number;
-};
-
-export namespace packet {
```

Comment:
I did not ask to remove this. I know some people use these to overcome the lack of some features on this library. Please revert.